### ISSUE TITLE
sassc failed to assets:precompile

### ISSUE THREAD

**Issue by u007** [galetahub/ckeditor#630]
hi,

im using ckeditor (4.1.5) and sassc (1.8.2)
when i deployed to server and does assets:precompile,
it failed..

please advice :)

**Comment by galetahub** [comment]
Hi,

Assets precompile can take a long time on simple servers. 
Try to use ckeditor (4.1.6) CDN support

**Comment by u007** [comment]
Hi,

When I do local precompile, it ends with error.
Is there a way to fix this or have to rely on CDN? CDN may cause another
request to server, especially js

**Comment by halilim** [comment]
I was also having problems with asset pre-compilation on Heroku. My app was exceeding the 50 MB intermediate asset cache there and was taking ~290 seconds on every push even when no assets were modified.

I've finally managed to get my asset pre-compilation time to ~27 seconds using [this config](https://gist.github.com/halilim/ddae5f77aea52f7d98fd). I think just setting `cdn_url` is not enough, I needed to exclude CKEditor assets from pre-compile list and set `run_on_precompile` to false. We might need to indicate this in the documentation.
